spec/features/hq/user_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
+
+RSpec.feature 'User CRUD spec:', :type => :feature do
+  
+  background do
+    i_sign_in_as_admin
+    an_user_exist 98, 'First User', 'first@example.com'
+    an_user_exist 99, 'Second User', 'second@example.com'
+  end
+  
+  scenario 'There should be a link to the Users page on the navbar' do
+    visit hq_root_path
+    expect(page).to have_link( 'Users', href: hq_users_path )
+  end
```

Comment:
you already tested this when you made the navigation integration tests: https://github.com/AgileVentures/osra/pull/387